algo-builder-tester/test/v5stake-test.ts

```diff
@@ -1,42 +1,54 @@
-import { AccountStore, Runtime, types as typesRT } from '@algo-builder/runtime';
-import { parsing, types as typesW } from '@algo-builder/web';
+import { AccountStore, Runtime, types as typesRT } from "@algo-builder/runtime";
+import { parsing, types as typesW } from "@algo-builder/web";
 
 import { LogicSigAccount } from "algosdk";
-import { assert } from 'chai';
+import { assert } from "chai";
```

Comment:
Consider avoid formatting changes in a feature PR, which makes it hard to see interesting part

Reply:
fixed in #8

Reply:
Yeah, next time let's only do formatting changes in separate PR. This time we don't need to revert formatting changes to make it neat